ComCompany/Lorry.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Text;
+
+namespace Company
+{
+    class Lorry : Car
+    {
+        public double CarryingCapacity { get; set; }
+        public override StringBuilder CarInformationToString()
```

Comment:
do you still need CarInformationToString() method?

Reply:
done